optuna/pruners/hyperband.py

```diff
@@ -50,6 +53,8 @@ class HyperbandPruner(BasePruner):
             :class:`~optuna.pruners.SuccessiveHalvingPruner`.
         n_brackets:
             The number of :class:`~optuna.pruners.SuccessiveHalvingPruner`\\ s (brackets).
+            For those who are curious why this defaults to :math`4`, see
```

Comment:
A nit, believing that we can afford to be more stiff and short here.
```suggestion
            Defaults to :math`4`. See
```